tests/views_integration/Tribe/Events/Views/V2/HTML/LoaderTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Tribe\Events\Views\V2\Views\HTML;
+
+use Tribe\Events\Views\V2\TestHtmlCase;
+
+class LoaderTest extends TestHtmlCase {
+
+	/**
+	 * @test
+	 */
+	public function it_should_contain_correct_html_classes() {
+		$template = $this->template->template( 'loader' );
+		$html = $this->document->html( $template );
+
+		$this->assertEquals(
+			$html->find( '.tribe-events-view-loader' )->count(),
```

Comment:
You use$html->find( '.tribe-events-view-loader' ) three times, I'd move it out to a variable?

Reply:
Not really because it breaks due to how it modifies the original object in some odd way. Still need to see why.